ui/components/app/edit-gas-fee-popover/network-status/index.scss

```diff
@@ -0,0 +1,41 @@
+.network-status {
+  margin: 24px 0 12px;
+
+  &__info {
+    border-top: 1px solid $ui-2;
+    border-bottom: 1px solid $ui-2;
+    height: 56px;
+    display: flex;
+    align-items: center;
+
+    &-separator {
```

Comment:
Sorry to be the BEM nazi 😂 but I'm wondering if these shouldn't be nested to avoid specificity issues and should be named:

* `.network-status__separator`
* `.network-status__field`
* `.network-status__field--priority-fee` (this one is good already)
* `.network-status__field-data`
* `.network-status__field-label`

Reply:
I appreciate your detailed reviews, I will take time to read BEM more. Thanks a lot.